packages/core/src/page.js

```diff
@@ -140,7 +140,7 @@ export class Page {
     execute,
     ...snapshot
   }) {
-    let { name, width, enableJavaScript } = snapshot;
+    let { name, width, enableJavaScript, disableShadowDom } = snapshot;
```

Comment:
* requires docs update https://docs.percy.io/docs/cli-configuration#per-snapshot-configuration which would be common for all the SDKs